HW_2/js/js_level1.js

```diff
@@ -0,0 +1,153 @@
+function consoleNumbers(n) {
+    if (n >= 0 && n <= 15) {
+        console.log(n++);
+        consoleNumbers(n);
+    }
+}
+
+console.log("Вывод числа до 15 рекурсией:");
+consoleNumbers(varSwitch);
+
+//А вот так в учебных целях через Switch, правда это на безумие похоже
+
+console.log("Вывод числа до 15 через Switch:");
+switch (varSwitch) {
+    case 0: console.log(0);
+    case 1: console.log(1);
+    case 2: console.log(2);
+    case 3: console.log(3);
+    case 4: console.log(4);
+    case 5: console.log(5);
+    case 6: console.log(6);
+    case 7: console.log(7);
+    case 8: console.log(8);
+    case 9: console.log(9);
+    case 10: console.log(10);
+    case 11: console.log(11);
+    case 12: console.log(12);
+    case 13: console.log(13);
+    case 14: console.log(14);
+    case 15: console.log(15);
+}
+
+//////////////////////////////////////////////////
+
+function fSummation(x,y) {
+    return parseInt(x) + parseInt(y);
+}
+
+function fMultiplication(x,y) {
+    return x * y;
+}
+
+function fSubtraction(x,y) {
+    return (x - y);
+}
+
+function fDivision(x,y) {
+    return (x / y).toFixed(2);
+}
+
+console.log("Результат арифмитических операций с переменными a и b (значения  " + a + " и " + b + "). Сложение, умножение, вычитание, деление соответственно:");
+console.log(fSummation(a, b));
+console.log(fMultiplication(a, b));
+console.log(fSubtraction(a, b));
+console.log(fDivision(a, b));
+
+//////////////////////////////////////////////////
+
+//Организация функции мат. расчетов.
+function mathOperation(arg1, arg2, operation) {
+    if (isNaN(arg1) || isNaN(arg2)) {
+        return "The parameters have to be numbers.";
+    }
+    else {
+        switch (operation) {
+            case "Summation":
+                return fSummation(arg1, arg2);
+            case "Multiplication":
+                return fMultiplication(arg1, arg2);
+            case "Subtraction":
+                return fSubtraction(arg1, arg2);
+            case "Division":
+                return fDivision(arg1, arg2);
+            default:
+                return "The calculation can't be done. Pls, check the operation type";
+        }
+    }
+}
+
+console.log(mathOperation("omg", 1, "Division"));
+console.log(mathOperation(2, 3, "gg wp"));
+console.log(mathOperation(2, 3, "Multiplication"));
+
+//////////////////////////////////////////////////
+
+//Сравнение даст false, ибо Null будет приведен к числу как 0+, а ноль так и есть точный 0.
+//Прикольная статья на эту тему: https://habr.com/ru/company/ruvds/blog/337732/
+var i = null, j = 0;
+console.log(i == j);
+
+/////////////////////////////////////////////////
+
+//Возведение в степень
+function power(val, pow) {
+    if (pow == 0) {
+        if (val >= 0) {
+            return 1;
+        } else {
+            return -1;
+        };
+    }
+    if (pow == 1) {
+        return val;
+    } else {
+        return val * power(val, pow - 1);
+    }
+};
+console.log("Возведение в степень:");
+var val,pow;
```

Comment:
1. Не забывайте ставить пробелы между операторами и операндами
```suggestion
var val, pow;
```
2. Не стоит создавать несколько переменных одним var. В JS чаще принят подход 1 var - 1 переменная